README-gke.md

```diff
@@ -30,7 +26,24 @@ It is also possible to use an existing cluster configuration directory, by expor
 
 Start the tool by running `./qserv-deploy.sh`
 
+Then get kubeconfig for your gke cluster, following example below:
+
+```
+gcloud auth login
+# Example:
+# PROJECT=neural-theory-215601
+PROJECT=<project>
+# Example:
+# CLUSTER=lsst-k8s-test
+CLUSTER=<cluster>
+gcloud config set project "$PROJECT"
+gcloud container clusters resize "$CLUSTER" --region us-central1-a --size=3
```

Comment:
This is also hard-coded, but in the limits of our current testing it should be fine. Once we are ready to spin up N number of k8s Qserv nodes we should clean this up.